app/src/main/res/layout-sw600dp/administrator_controls_account_actions_view.xml

```diff
@@ -18,14 +18,13 @@
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
```

Comment:
`android:layout_height="wrap_content"` 

Remove this considering that it is already part of the styles.xml

Check:
`Caption -> TextView.Common1`

Also, reformat the code. https://github.com/oppia/oppia-android/wiki/XML-code-style-format#beyond-setting-changes